<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="1" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-07-04T19:45:28 IST" name="Suite" finished-at="2025-07-04T19:47:07 IST" duration-ms="98787">
    <groups>
    </groups>
    <test started-at="2025-07-04T19:45:28 IST" name="Test" finished-at="2025-07-04T19:47:07 IST" duration-ms="98787">
      <class name="testCases.TC3_DataDrivenLogin">
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:45:28 IST" name="setUp" finished-at="2025-07-04T19:45:37 IST" duration-ms="9209" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:45:39 IST" name="setUp" finished-at="2025-07-04T19:45:46 IST" duration-ms="6258" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="VerigyDatadrivenLoginFeature(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:45:46 IST" name="VerigyDatadrivenLoginFeature" data-provider="LoginData" finished-at="2025-07-04T19:46:02 IST" duration-ms="16844" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[test123@yopmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[An exception occured...no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']"}
  (Session info: chrome=137.0.7151.120)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.28.0', revision: 'ac342546e9'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '15.5', java.version: '23.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [0a75b6a9bdb3bc2ef98cde8c9340b1e0, findElement {using=xpath, value=//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.120, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: /var/folders/w9/37hb8zlj6gl...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:64998}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: mac, proxy: Proxy(), se:cdp: ws://localhost:64998/devtoo..., se:cdpVersion: 137.0.7151.120, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 0a75b6a9bdb3bc2ef98cde8c9340b1e0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: An exception occured...no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']"}
  (Session info: chrome=137.0.7151.120)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.28.0', revision: 'ac342546e9'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '15.5', java.version: '23.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [0a75b6a9bdb3bc2ef98cde8c9340b1e0, findElement {using=xpath, value=//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.120, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: /var/folders/w9/37hb8zlj6gl...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:64998}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: mac, proxy: Proxy(), se:cdp: ws://localhost:64998/devtoo..., se:cdpVersion: 137.0.7151.120, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 0a75b6a9bdb3bc2ef98cde8c9340b1e0
at org.testng.Assert.fail(Assert.java:111)
at testCases.TC3_DataDrivenLogin.VerigyDatadrivenLoginFeature(TC3_DataDrivenLogin.java:76)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerigyDatadrivenLoginFeature -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:02 IST" name="tearDown" finished-at="2025-07-04T19:46:03 IST" duration-ms="464" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:03 IST" name="setUp" finished-at="2025-07-04T19:46:10 IST" duration-ms="7068" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="VerigyDatadrivenLoginFeature(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:10 IST" name="VerigyDatadrivenLoginFeature" data-provider="LoginData" finished-at="2025-07-04T19:46:14 IST" duration-ms="3932" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[meryl.doe@yopmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[valid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testCases.TC3_DataDrivenLogin.VerigyDatadrivenLoginFeature(TC3_DataDrivenLogin.java:64)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerigyDatadrivenLoginFeature -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:14 IST" name="tearDown" finished-at="2025-07-04T19:46:14 IST" duration-ms="338" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:14 IST" name="setUp" finished-at="2025-07-04T19:46:21 IST" duration-ms="6881" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="VerigyDatadrivenLoginFeature(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:21 IST" name="VerigyDatadrivenLoginFeature" data-provider="LoginData" finished-at="2025-07-04T19:46:34 IST" duration-ms="12481" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[deryl.doe@yopmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[deryl]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[An exception occured...no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']"}
  (Session info: chrome=137.0.7151.120)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.28.0', revision: 'ac342546e9'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '15.5', java.version: '23.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [47883ad5bb2115dbfc74361ae0deb02c, findElement {using=xpath, value=//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.120, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: /var/folders/w9/37hb8zlj6gl...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:65226}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: mac, proxy: Proxy(), se:cdp: ws://localhost:65226/devtoo..., se:cdpVersion: 137.0.7151.120, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 47883ad5bb2115dbfc74361ae0deb02c]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: An exception occured...no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']"}
  (Session info: chrome=137.0.7151.120)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.28.0', revision: 'ac342546e9'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '15.5', java.version: '23.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [47883ad5bb2115dbfc74361ae0deb02c, findElement {using=xpath, value=//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.120, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: /var/folders/w9/37hb8zlj6gl...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:65226}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: mac, proxy: Proxy(), se:cdp: ws://localhost:65226/devtoo..., se:cdpVersion: 137.0.7151.120, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 47883ad5bb2115dbfc74361ae0deb02c
at org.testng.Assert.fail(Assert.java:111)
at testCases.TC3_DataDrivenLogin.VerigyDatadrivenLoginFeature(TC3_DataDrivenLogin.java:76)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerigyDatadrivenLoginFeature -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:34 IST" name="tearDown" finished-at="2025-07-04T19:46:34 IST" duration-ms="439" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:34 IST" name="setUp" finished-at="2025-07-04T19:46:41 IST" duration-ms="7362" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="VerigyDatadrivenLoginFeature(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:41 IST" name="VerigyDatadrivenLoginFeature" data-provider="LoginData" finished-at="2025-07-04T19:46:54 IST" duration-ms="12516" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[chethan@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[xyz]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[An exception occured...no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']"}
  (Session info: chrome=137.0.7151.120)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.28.0', revision: 'ac342546e9'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '15.5', java.version: '23.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [cba991f8fa75c037f655be1550ef0355, findElement {using=xpath, value=//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.120, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: /var/folders/w9/37hb8zlj6gl...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:65342}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: mac, proxy: Proxy(), se:cdp: ws://localhost:65342/devtoo..., se:cdpVersion: 137.0.7151.120, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: cba991f8fa75c037f655be1550ef0355]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: An exception occured...no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']"}
  (Session info: chrome=137.0.7151.120)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.28.0', revision: 'ac342546e9'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '15.5', java.version: '23.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [cba991f8fa75c037f655be1550ef0355, findElement {using=xpath, value=//ul[@class='nav navbar-nav']//li//a[text()=' Logged in as ']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.120, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: /var/folders/w9/37hb8zlj6gl...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:65342}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: mac, proxy: Proxy(), se:cdp: ws://localhost:65342/devtoo..., se:cdpVersion: 137.0.7151.120, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: cba991f8fa75c037f655be1550ef0355
at org.testng.Assert.fail(Assert.java:111)
at testCases.TC3_DataDrivenLogin.VerigyDatadrivenLoginFeature(TC3_DataDrivenLogin.java:76)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerigyDatadrivenLoginFeature -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:54 IST" name="tearDown" finished-at="2025-07-04T19:46:54 IST" duration-ms="420" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:46:54 IST" name="setUp" finished-at="2025-07-04T19:47:01 IST" duration-ms="6930" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="VerigyDatadrivenLoginFeature(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:47:01 IST" name="VerigyDatadrivenLoginFeature" data-provider="LoginData" finished-at="2025-07-04T19:47:07 IST" duration-ms="5344" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[maggie@yopmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test@1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerigyDatadrivenLoginFeature -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:47:07 IST" name="tearDown" finished-at="2025-07-04T19:47:07 IST" duration-ms="489" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC3_DataDrivenLogin@327514f]" started-at="2025-07-04T19:47:07 IST" name="tearDown" finished-at="2025-07-04T19:47:07 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC3_DataDrivenLogin -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
